src/megachatapi_impl.cpp

```diff
@@ -1175,6 +1175,44 @@ int MegaChatApiImpl::getInitState()
     return initState;
 }
 
+bool MegaChatApiImpl::enableChat(const char *sid)
+{
+    if (sid == NULL)
+    {
+        API_LOG_ERROR("Failure to enable chat, invalid session");
+        return false;
+    }
+
```

Comment:
If init state is already a valid one, we can safely return true (chat is enabled and working)

Reply:
Reject this change, better to let the app know that they are calling `enableChat()` when already initialized